libs/dh/admin/feature-create-user-role/src/lib/dh-create-user-role-modal.component.html

```diff
@@ -79,15 +68,6 @@
       [previousButtonLabel]="t('steps.masterData.label')"
       [nextButtonLabel]="t('save')"
     >
-      <watt-field-error
-        class="permissions-error"
-        *ngIf="
-          (isSubmitted || selectedPermissions.touched) && selectedPermissions.hasError('required')
-        "
-      >
-        {{ "shared.fieldRequiredErrorText" | transloco }}
-      </watt-field-error>
-
```

Comment:
suggestion: it seems like this error is still relevant. It's shown on top of the Permissions table, and appears when the stepper is submitted without assigning a permission.